app/pangram.hs

```diff
@@ -0,0 +1,23 @@
+--http://exercism.io/exercises/haskell/pangram/readme
```

Comment:
* first change your logic such that it doesn't require you to maintain separate lists for uppercase and lowercase characters.
* Next, add another solution which uses the hint given at http://www.vacationlabs.com/haskell/basic-types-and-functions.html#pangram